m4_sessions/1803-inning/group_5.md

```diff
@@ -2,12 +2,34 @@ Group 5
 
 Group Members:
 
+Bruce Chung
+Alex Barnes
+Nico Lewis
+
 Name of Session:
 
+Social Media and Fake News
+
 Session Description (Why should students come to this session):
 
+Social media is already the main source of news for this generation.  Unlike tranditional news outlets, social media allows news to propogate on attention alone without regard to journalistic accuracy.  This is becoming an increasingly widespread and serious problem, even to national security as evidenced by the proliferation of Russian Fake News on facebook during the 2016 Election.
+
 Session Outcomes (What will students learn/leave with?):
 
+1) Understanding of what fake news is
+2) How fake news impacts our society
+3) How it is a 21st century problem and potential methods to deal with it
+4) Ability to spot fake news
+5) Journalistic spin vs fake news
+
 Session Outline (Please include who is facilitating and how much time you are going to spend in each section):
 
+Intro/Icebreaker (10 min)
```

Comment:
Excellent outline, be sure to add who is facilitating which part and narrow down your resources!